widgets/show-dataviz-widget.php

```diff
@@ -48,12 +48,19 @@ public function widget( $args, $instance ) {
  public function form( $instance ) {
   // outputs the options form on admin
   $id = ! empty( $instance['id'] ) ? __( $instance['id'], 'wp-odm_dash') : null;
-
+  $hide_desc = ! empty( $instance['hide_description'] ) ? __( $instance['hide_description'], 'wp-odm_dash') : "false";
```

Comment:
@DragonVirus value (true/false) should be boolean, not string. Suggestion:

$hide_desc = isset( $instance['hide_description'] ) ? $instance['hide_description'] : false;